spokestack/asr/spokestack/cloud_client.py

```diff
@@ -13,17 +13,17 @@
 
 
 class CloudClient:
-    """ Spokestack client for cloud based speech to text
+    """Spokestack client for cloud based speech to text
 
-        Args:
```

Comment:
for these mass changes to the formatter, we will want to create a .git-blame-ignore-revs file so that they are ignored when viewing the history
https://akrabat.com/ignoring-revisions-with-git-blame/

Reply:
can we do this with the documentation html stuff also?

Reply:
it doesn't matter for generated things like documentation
we just want to preserve readable diffs/blames from sweeping changes for things that are maintained/viewed manually

Reply:
that said, we would still want the generated documentation stuff on its own commits, probably in the same commit that updates the release number

Reply:
ok that makes sense. I'll do my feature commit and then another one with the docs build and version change.